pictures/README.md

```diff
@@ -1 +1,18 @@
 Use this directory for large pictures
+
```

Comment:
Not part of this PR but can you maybe change the typo in the global README  is currenctly under dev to currently?